lib/node_modules/@stdlib/string/next-grapheme-cluster-break/docs/repl.txt

```diff
@@ -0,0 +1,31 @@
+
+{{alias}}( str, idx )
```

Comment:
There is some inconsistency with the naming conventions for the second parameter. Here, you use `idx`, but in the README, you use `position`. Would be good to standardize. In the original [RFC](https://github.com/stdlib-js/stdlib/issues/357), it was `fromIndex`, which, I believe, has origins in the ECMAScript spec for various String#prototype methods.